lib/components/user/monitored-trip/trip-monitored-days.tsx

```diff
@@ -0,0 +1,81 @@
+import { FormattedList, FormattedMessage, useIntl } from 'react-intl'
+
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+import React from 'react'
+
+import { ALL_DAYS } from '../../../util/monitored-trip'
+import { getBaseColor } from '../../util/base-color'
+import FormattedDayOfWeek from '../../util/formatted-day-of-week'
+import FormattedDayOfWeekCompact from '../../util/formatted-day-of-week-compact'
+
+import styled from 'styled-components'
+
+interface Props {
+  days: string[]
+}
+
+const DayCircleContainer = styled.div`
+  display: flex;
+  gap: 4px;
+`
+
+const MonitoredDayCircle = styled.span<{
+  baseColor: string
+  monitored: boolean
+}>`
+  align-items: center;
+  background-color: ${(props) =>
+    props.monitored ? props.baseColor : 'transparent'};
```

Comment:
```suggestion
    props.monitored ? props.baseColor : '#004f9611;'};
```
Thoughts?

Reply:
I went with `#ECECEC` so we can replace with `grey[50]` when building-blocks (finally) gets merged

Reply:
They key bit here is the opacity though

Reply:
I went with `20` opacity, I found `11` pretty hard to see!

Reply:
11 is so light, it's hard for me to distinguish it as a circle. It makes me feel like I'm failing a test at the optometrist. @binh-dam-ibigroup  can you break this tie

| 11 | 20 |
|--------|-------|
| ![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/f16c0954-3515-4717-a5d7-e4e46f3020da) | ![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/9327de58-0d5d-4c73-a98c-a97ff2b25650) |

Reply:
Sounds good let's leave it as is